Library/Homebrew/extend/os/linux/gnu_tar.rb

```diff
@@ -0,0 +1,14 @@
+module UnpackStrategy
```

Comment:
Please move this to a new file `extend/os/linux/unpack_strategy/tar`

Reply:
I'll see now